eng/common/scripts/Validate-Package.ps1

```diff
@@ -212,10 +212,14 @@ $changeLogStatus = [PSCustomObject]@{
 ValidateChangeLog $changeLogPath $versionString $changeLogStatus
 
 # API review and package name validation
-$apireviewDetails = VerifyAPIReview $PackageName $pkgInfo.Version $Language
+$fulPackageName = $pkgName
+if ($pkgInfo.Group){
```

Comment:
This will fail in strict mode. You need to check if this property exists. Something similar to https://github.com/Azure/azure-sdk/blob/main/eng/scripts/PackageList-Helpers.ps1#L259